outputs.tf

```diff
@@ -1,9 +1,14 @@
 output "ecr_pullthroughcache_policy_arn" {
   value       = aws_iam_policy.ecr_pullthroughcache.arn
-  description = "The name of the resource."
+  description = "The ARN of the ECR pull-through cache policy."
 }
 
 output "ecr_pullthroughcache_policy_name" {
   value       = aws_iam_policy.ecr_pullthroughcache.name
-  description = "The name of the resource."
+  description = "The name of the ECR pull-through cache policy."
+}
+
+output "upstream_repository_uri" {
```

Comment:
I would prefer to use something `ecr_pullthroughcache_repository_uri`, the upstream repository is the other one.